blog/2024/07-30-okd-pre-release-testing.md

```diff
@@ -1,6 +1,7 @@
 ---
 draft: false 
 date: 2024-07-30
+authors: ["jaimemagiera"]
```

Comment:
Don't want to nitpick too much, but does this really belong in this patch?

Reply:
It's a fair flag and you are right that strictly it should be in a separate changeset.

Given it's low-risk and a minor change I won't pull it out for now